integration/README.md

```diff
@@ -18,10 +18,19 @@ Additionally, ~/.geopmrc should have the following contents (at a minimum):
 ```
 #!/bin/bash
 
+# Path to the GEOPM source code
 export GEOPM_SOURCE=${HOME}/geopm
+
+# Path to the installation artifacts for GEOPM
 export GEOPM_INSTALL=${HOME}/build/geopm
+
+# Path to for job output
```

Comment:
maybe "path to location of job output directories"?

Reply:
Doh.  Yes, that's much better.